### DISCUSSION TITLE
sql console plugin

### DISCUSSION THREAD

**Discussion by EurekaJiang** [dbeaver/dbeaver#18954] [Ideas]
how about add a sql  assistant （like use chatgpt），help user with sql sentence

**Comment by E1izabeth** [comment]
We have already done it in 22.3.4 https://github.com/dbeaver/dbeaver/releases

**Reply by DarylMarcapura** [reply]
how can i install gpt3?

**Reply by EurekaJiang** [reply]
just click the link

**Reply by EurekaJiang** [reply]
> We have already done it in 22.3.4 https://github.com/dbeaver/dbeaver/releases

I just downloaded  it，but it seems that I have no idea how to use the feature

**Reply by E1izabeth** [reply]
Wiki and video are on the way.

**Reply by E1izabeth** [reply]
We made it https://dbeaver.com/docs/wiki/OpenAI-SQL-Assistance/

**Reply by EurekaJiang** [reply]
> We made it https://dbeaver.com/docs/wiki/OpenAI-SQL-Assistance/

<img width="889" alt="image" src="https://user-images.githubusercontent.com/41953750/217828626-05f8a5a3-1ec0-4724-b177-58f5919593ea.png">
I couldn't see where the ChatGPT icon is  in version 22.3.4

**Reply by E1izabeth** [reply]
Have you installed the plugin?

**Reply by EurekaJiang** [reply]
<img width="726" alt="image" src="https://user-images.githubusercontent.com/41953750/217992926-be89b7ff-4a65-472c-a67d-0d980a528433.png">
I have some problems when installed the plugin.Is there another way to installed the plugin?

**Reply by E1izabeth** [reply]
I doubt why it could happen, but you may need to uninstall dbeaver completely and install again.
